scripts/generate-analysis-files-for-release.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+set -e
+set -o pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
+# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
+# local testing running/testing of all other steps
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+
+analyses_dir="$BASEDIR/analyses"
+data_dir="$BASEDIR/data"
+scratch_dir="$BASEDIR/scratch"
+
+# Collapsed RNA-seq files
+echo "Create collapse RSEM files"
+bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh
+
+# Create the independent sample list using the *BASE* histology file
+echo "Create independent sample list"
+OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/independent-samples/run-independent-samples.sh
+
+# Fusion filtering
+echo "Create fusion filtered list"
+OPENPBTA_BASE_RELEASE=1 bash ${analyses_dir}/fusion_filtering/run_fusion_merged.sh
+
+# Fusion summary
+echo "Run fusion summary for subtypes"
+bash ${analyses_dir}/fusion-summary/run-new-analysis.sh
+
+# Compile all the files that need to be included in the release in one place
+# in the scratch directory
+compiled_dir=${scratch_dir}/analysis_files_for_release
+mkdir -p ${compiled_dir}
```

Comment:
I would move these lines up before you start running modules, right after other directories are defined in lines 19-21